unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Skybox3DElements/SkyboxElements.cs

```diff
@@ -13,11 +12,11 @@ public class SkyboxElements : IDisposable
         public SkyboxPlanarElements planarElements;
         public SkyboxElementsReferences references;
 
-        public SkyboxElements()
+        public async Task Initialize(IAddressableResourceProvider addresableResolver, MaterialReferenceContainer materialReferenceContainer)
```

Comment:
@mikhail-dcl , here as well. Should it be `Task` or `UniTask` with `cancelationToken`?

Reply:
Changed to Unitask